state-chain/pallets/cf-elections/src/mock.rs

```diff
@@ -102,6 +102,9 @@ pub fn election_test_ext(test_setup: TestSetup) -> TestRunner<TestContext> {
 			}
 			MockEpochInfo::next_epoch(test_setup.all_authorities());
 
+			Pallet::<Test, _>::do_try_state()
```

Comment:
this will only run when we start up the test, would be good to have this run at least on the end of each test too

Reply:
as discussed with Dan we will create a separate issue to properly handle this and make it available for all pallets